qiskit/visualization/pulse/matplotlib.py

```diff
@@ -657,8 +658,9 @@ def _draw_framechanges(self, ax, fcs: Dict[int, ShiftPhase], y0: float) -> bool:
                 ha="center",
                 va="center",
             )
+        # TODO: doesn't return anyting?
```

Comment:
Not sure is there is return missing or wrong type annotation

Reply:
Based on the method usage, it seems like a wrong type annotation. It should be `None`.